scripts/workers/submission_worker.py

```diff
@@ -591,6 +591,7 @@ def process_add_challenge_message(message):
         challenge = Challenge.objects.get(id=challenge_id)
     except Challenge.DoesNotExist:
         logger.exception("Challenge {} does not exist".format(challenge_id))
+        raise
```

Comment:
@nsjcorps I don't think this change should be the part of this PR. It might break things in production. For now let's only add tests in this PR. If we need to raise a exception you can create a separate PR for that

Reply:
Can you remove this